### ISSUE TITLE
[Bug] The `RefreshTokenValidity` max value is too low for `AWS::Cognito::UserPoolClient`

### ISSUE THREAD

**Issue by BR0kEN-** [aws-cloudformation/cfn-lint#3566]
### CloudFormation Lint Version

1.9.3

### What operating system are you using?

Mac

### Describe the bug

The max value for the `AWS::Cognito::UserPoolClient#RefreshTokenValidity` property is `315360000` as per https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpoolclient.html#cfn-cognito-userpoolclient-refreshtokenvalidity

Though the present schemas set it to `3650` which causes `E3034 86400 is greater than the maximum of 3650`.

### Expected behavior

The `AWS::Cognito::UserPoolClient#RefreshTokenValidity` max value is `315360000`.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09

Resources:
  UserPoolClient:
    Type: AWS::Cognito::UserPoolClient
    Properties:
      ClientName: cognito-client
      UserPoolId: CognitoUserPoolId
      AllowedOAuthFlowsUserPoolClient: true
      GenerateSecret: true
      TokenValidityUnits:
        AccessToken: seconds
        IdToken: seconds
        RefreshToken: seconds
      AccessTokenValidity: 3600
      IdTokenValidity: 3600
      RefreshTokenValidity: 86400
      AllowedOAuthScopes:
        - openid
        - profile
      CallbackURLs:
        - https://site.com/api/auth/cognito
      LogoutURLs:
        - https://site.com/api/auth/cognito/post-logout
```
